proposals/utf8-string-literals.md

```diff
@@ -154,6 +119,54 @@ It seems unlikely that we would regret the target type conversion between string
 
 It seems more likely that we'd regret the `u8` suffix pointing to `ReadOnlySpan<byte>` instead of `Utf8String`. It would be similar to how we regret that `stackalloc int[]` has a natural type of `int*` instead of `Span<int>`. This is not a deal breaker though, just an inconvenience.
 
+### Conversions between `string` constants and `byte` sequences
```

Comment:
I'm a little confused by this section: is this part of the feature still happening? Or was it removed?

Reply:
nit: I'd probably just leave a one sentence note in alternatives ("We experimented with a conversion ... but decided against it"), but wouldn't keep all this detail. I got a bit confused too ;-)